docs/app/Examples/modules/Checkbox/States/Checked.js

```diff
@@ -1,18 +1,8 @@
-import React, {Component} from 'react';
+import React from 'react';
 import {Checkbox} from 'stardust';
 
-export default class CheckboxCheckedExample extends Component {
-  state = {isChecked: true};
-
-  handleChange = e => {
-    this.setState({
-      isChecked: !this.state.isChecked
-    });
-  };
-
-  render() {
-    return (
-      <Checkbox label='This checkbox comes prechecked' checked={this.state.isChecked} onChange={this.handleChange} />
-    );
-  }
-}
+export default CheckboxCheckedExample => {
+  return (
+    <Checkbox defaultChecked label='This checkbox comes prechecked' />
+  );
+};
```

Comment:
To check a checkbox, just set the React `defaultChecked` prop.  This works because props are spread where they should be.